clients/privacy-center/components/Card.tsx

```diff
@@ -28,11 +30,11 @@ const Card: React.FC<CardProps> = ({
     userSelect="none"
     m={2}
     _hover={{
-      boxShadow: "complimentary-2xl",
```

Comment:
This shadow provided a purplish-trim by default, which must have been from old privacy center styling. It makes providing custom colours a real chore, so I've swapped this out with a nice box shadow from a recent Figma.